firmware/integration/rusefi_config.txt

```diff
@@ -1561,7 +1561,7 @@ uint8_t alsEtbPosition;;"", 1, 0, 0, 20000, 0
 	pin_input_mode_e ALSActivatePinMode;
 
 	uint8_t autoscale tpsSecondaryMaximum;For Ford TPS, use 53%. For Toyota ETCS-i, use 65%;"%", 0.5, 0, 0, 100, 1
```

Comment:
I use 66%, but 64.5 - 66.5 or so seems to work w/o blowing the discrepancy (though further from "perfect" means bigger step at partial switch-over).  Irrelevant w/ "monitored" mode.